src/pages/plugins/tutorials/udt-deep-dive/plugin-workflows.md

```diff
@@ -0,0 +1,74 @@
+---
+keywords:
+  - Creative Cloud
+  - API Documentation
+  - UXP
+  - Plugins
+  - JavaScript
+  - ExtendScript
+  - SDK
+  - C++
+  - Scripting
+title: Plugin workflows
+description:
+---
+
+# Plugin workflows
+
+Once you've added a plugin to your developer workspace, there are several things you can do with it. All of the following features are accessed via the **•••**/**Actions** dropdown associated with each plugin.
+
+## Load plugin
+
+Plugins added to the developer workspace do not automatically get added to their supported host application. Instead, once you launch the host application, you'll be able to tell the plugin (**••• > Load**) to launch in the host. This allows you to use the marketplace version of your plugin while you're not doing active development, but load up the development version when you need to make a new version of the plugin.
+
+Plugins that load successfully will show a small green notification at the bottom of the screen indicating that the plugin was loaded. 
+
+![Load Successful](./images/udt-load-successful.png)
+
+If a plugin _fails_ to load, a small red notification will appear with a **Details** link.
+
+![Load Failed](./images/udt-load-failed.png)
+
+Clicking the `Details` link in this message will show an error log indicating why the plugin failed to load:
+
+![Failure Log](./images/udt-failure-log.png)
+
+Once your plugin is loaded, you'll typicaly use the **Reload** action if you need to see any changes in your plugin's behavior. However, if you make changes to your plugin's `manifest.json` file, you'll want to use the **Load** action to load the plugin from scratch.
+
+
+## Watching and Reloading
+
+Selecting the `Watch` item from the Actions menu causes the Developer Tool to watch for any changes in your code on disk. 
+Whenever you make a change to a file in your project, UDT will reload your plugin automatically. This makes for a very rapid code->make mistakes->fix mistakes->reload cycle.
+
+You can also reload your plugin manually by selecting `Reload` from the Actions menu. If you have `Watch` enabled, you shouldn't have to do this. But some people are superstitious.
+
+However, for manifest changes, this action does _not_ refresh the changes. For that you should 'Unload' and 'Load' the plugin again.
+
+## Debug your plugin
+
+You can also debug your plugin by choosing `Debug` from the Actions menu. This brings up a window like this:
+
+![Debug Window](./images/udt-debugger.png)
+
+In this debugger (which resembles the Chrome Developer tool) you can look at the console log, and do the usual debugger things such as setting breakpoints, stepping into and out of functions, etc. 
+
+The tool does have several limitations including the inability to edit CSS styles in the property inspector, but even so, you can now walk through code, inspect elements, and more. <!-- TODO Verify about CSS-->
```

Comment:
@kareemmyk  Is this still true about editing CSS?

Reply:
I will get back to you on this

Reply:
@padmkris123 The editing of css is still possible. So we should remove ```limitations including the inability to edit CSS styles in the property inspector,```